src/dps-email-worker/src/main/resources/application.properties

```diff
@@ -1,3 +1,4 @@
+server.port=8085
```

Comment:
Do we want to consider setting this a variable. Since we have to run multiple projects?

Reply:
thanks